src/tabs/classes.scss

```diff
@@ -2,18 +2,30 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
 
-.iui-tabs-horizontal {
-  @include iui-tabs-horizontal;
+.iui-tabs-wrapper {
```

Comment:
This class is only needed for the stripe that can be placed inside `iui-tabs`.

Reply:
`iui-tabs` is a list, so placing stripe inside would not make sense and could mess up `:last-child`